<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>allEvents</class>
 <widget class="QMainWindow" name="allEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>current_events</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Sort 1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Delete Event</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Update Event</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Sort 2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Delete History</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>Event</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Place</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Event's Start Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Event's End Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reminder Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Done</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
